src/server_manager/cloud/gcp_api.ts

```diff
@@ -135,51 +191,43 @@ export class RestApiClient {
    *
    * @see https://cloud.google.com/compute/docs/reference/rest/v1/instances/insert
    *
-   * @param projectId - The GCP project ID.
-   * @param zoneId - The zone in which to create the instance.
+   * @param scope - Indicates the GCP project and zone.
    * @param data - Request body data. See documentation.
    */
-  async createInstance(projectId: string, zoneId: string, data: {}): Promise<ComputeEngineOperation> {
+  async createInstance(scope: ZoneScope, data: {}): Promise<ComputeEngineOperation> {
     const operation = await this.fetchAuthenticated<ComputeEngineOperation>(
         'POST',
-        new URL(`https://compute.googleapis.com/compute/v1/projects/${projectId}/zones/${
-            zoneId}/instances`),
+        new URL(`${zoneLink(scope)}/instances`),
         this.GCP_HEADERS, null, data);
-    return await this.computeEngineOperationZoneWait(projectId, zoneId, operation.name);
+    return await this.computeEngineOperationZoneWait(scope, operation.name);
```

Comment:
`scope` is ambiguous, since there are region scopes too. Use more explicit names.  `zoneLocator` or `zone` should be fine.

Same everywhere else where you just use `scope`

Reply:
Done